<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandLine</class>
 <widget class="QDockWidget" name="CommandLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>232</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Command Window</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="cmdHist">
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="cmdEntry"/>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdEntry</sender>
   <signal>returnPressed()</signal>
   <receiver>runButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
